currencyConverter.js

```diff
@@ -0,0 +1,57 @@
+var request = require('request');
+var cheerio = require('cheerio');
+
+/**
+* Rounds off currency output 2 decimal places
+*/
+
+var roundOff = function (currency) {
+
+  return Math.round(currency * 100) / 100;
+
+}
+
+/**
+* Converts two parameters from param to param1 eg: USD to ZAR
+* @param fromCurrency selects the currency you would like to convert
+* @param toCurrency converts fromCurrency to new currency user is requesting
+*/
+
+var getCurrency = function (fromCurrency, toCurrency) {
+
+  var url = 'http://themoneyconverter.com/'+fromCurrency+'/'+toCurrency+'.aspx'
+
+  return sendRequest(url);	
+
+}
+
+/**
+* Sends request to currency url with user params
+* @currencyUrl string returns the function getCurrency url
+*/
+
+var sendRequest = function (currencyUrl) {
+
+  request({ url: currencyUrl }, function (err, response, body) {
+    if (err) {
+      console.log('Error: ', err);
+    }
+
+    var $ = cheerio.load(body);
+    var currency = $('div.tmc-well.switch-table > p > b').text();
+	  
+    console.log(roundOff(currency));
+
+    })
+}
+
+var returnResult = function (fromCurrency, toCurrency) {
```

Comment:
This isn't necessary. The `getCurrency` method should take a callback that gets used when you get a result. 

In the end, your program code would look like the inner block here.

Something like this:

```js
getCurrency('zar', 'usd', function (error, result) {
  if (error) {
    console.error("Error: " + error);
  } else {
    console.log("Converted: " + result);
  }
});
```

This way, the client code (the code that is using this file) can choose what to do with the result. My code there console.logs it, but others might use it in a calculation or something. Have a read about callbacks in JS